src/dpp/snowflake.cpp

```diff
@@ -20,73 +20,28 @@
  ************************************************************************************/
 #include <dpp/snowflake.h>
 #include <dpp/json.h>
+#include <charconv>
+#include <string>
 
 namespace dpp {
 
-snowflake::snowflake(const uint64_t &value) : value(value) {}
-
-snowflake::snowflake(const std::string &string_value) {
-	try {
-		value = std::stoull(string_value);
-	}
-	catch (const std::exception &) {
+snowflake::snowflake(std::string_view string_value) noexcept {
```

Comment:
All of this inside `namespace dpp {` should be indented as [defined by the code convention](https://dpp.dev/coding-standards.html#:~:text=all%20code%20blocks%20delimited%20within%20curly%20braces%20should%20be%20indented%20neatly%20and%20uniformly.).

Reply:
we don't indent the dpp namespace as it would just result in the entire file being indented 8 chars to the right...

Reply:
(this should perhaps be added to the coding style)

Reply:
Then utility, and potentially more files, needs rewriting.

Reply:
I think you misunderstood we don't and shouldn't indent these namespaces as they just push the entire file to the right

Reply:
Agreed upon that the style guide needs to be changed to state that this case is correct.